app/controllers/app.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+require './lib/peeps'
+require 'pg'
+require_relative 'user_controller'
+
+class Chitter < Sinatra::Base 
+  use UserController
+  enable :sessions, :method_override
+
+  get '/' do
+    erb :"index"
+  end
+
+  get '/peeps' do
+    @peeps = Peeps.all
+    erb :"peeps/peeps"
+  end
+
+  get '/peeps/new' do
+    erb :"peeps/new"
+  end
+
+  post '/peeps' do
```

Comment:
really great Controller here - very clear and clear responsibilities and paths throughout